analyzers/src/SonarAnalyzer.CSharp/Rules/LossOfFractionInDivision.cs

```diff
@@ -47,8 +47,7 @@ public sealed class LossOfFractionInDivision : SonarDiagnosticAnalyzer
                 {
                     var division = (BinaryExpressionSyntax)c.Node;
 
-                    if (c.SemanticModel.GetSymbolInfo(division).Symbol is not IMethodSymbol symbol
-                        || symbol.ContainingType is null
```

Comment:
I don't see how the `ContainingType` can be null here - UTs never reach this part and ITs don't fail. 
We should be safe IMO.

Same for checking if the symbol is not IMethodSymbol.